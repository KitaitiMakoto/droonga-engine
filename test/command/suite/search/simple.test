#@include fixture/documents.jsons
{
  "type": "search",
  "dataset": "Droonga",
  "body": {
    "queries": {
      "result": {
        "source": "Sections",
        "output": {
          "elements": [
            "startTime",
            "elapsedTime",
            "count",
            "attributes",
            "records"
          ],
          "limit": -1,
          "attributes": ["_key", "title"]
        }
      }
    }
  }
}
