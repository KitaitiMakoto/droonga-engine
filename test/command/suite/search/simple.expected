{
  "inReplyTo": "request-id",
  "statusCode": 200,
  "type": "search.result",
  "body": {
    "result": {
      "count": 9,
      "records": [
        [
          "1.1",
          "Groonga overview"
        ],
        [
          "1.2",
          "Full text search and Instant update"
        ],
        [
          "1.3",
          "Column store and aggregate query"
        ],
        [
          "1.4",
          "Inverted index and tokenizer"
        ],
        [
          "1.5",
          "Sharable storage and read lock-free"
        ],
        [
          "1.6",
          "Geo-location (latitude and longitude) search"
        ],
        [
          "1.7",
          "Groonga library"
        ],
        [
          "1.8",
          "Groonga server"
        ],
        [
          "1.9",
          "Groonga storage engine"
        ]
      ]
    }
  }
}
